frontend/javascripts/navbar.tsx

```diff
@@ -345,10 +350,22 @@ function HelpSubMenu({
           Keyboard Shortcuts
         </a>
       </Menu.Item>
+      {isAuthenticated ? (
+        <>
+          <Menu.Item key="get_help" onClick={() => setIsHelpModalOpen(true)}>
+            Get help / Ask a question
```

Comment:
```suggestion
            Get help / Ask a question
```

I'm not really fond of the `/`. "Ask a question" alone should cover it pretty well, no?

Reply:
Fair enough. I shortened it to just "Ask a question"